pkg/cmd/server/origin/run_components.go

```diff
@@ -291,7 +291,9 @@ func (c *MasterConfig) RunBuildPodController() {
 func (c *MasterConfig) RunBuildImageChangeTriggerController() {
 	bcClient, _ := c.BuildImageChangeTriggerControllerClients()
 	bcInstantiator := buildclient.NewOSClientBuildConfigInstantiatorClient(bcClient)
-	factory := buildcontrollerfactory.ImageChangeControllerFactory{Client: bcClient, BuildConfigInstantiator: bcInstantiator}
+	bcIndex := &oscache.StoreToBuildConfigListerImpl{c.Informers.BuildConfigs().Indexer()}
+	bcIndexSynced := c.Informers.BuildConfigs().Informer().HasSynced
```

Comment:
[This method](https://github.com/openshift/origin/blob/e9081aee47f71af8faeced24e991a6afcab2f12d/pkg/controller/shared/shared_informer.go#L81) starts all shared informers and it's called in master [here](https://github.com/openshift/origin/blob/e9081aee47f71af8faeced24e991a6afcab2f12d/pkg/cmd/server/start/start_master.go#L419).